justspacesapp/templates/edit_survey.html

```diff
@@ -0,0 +1,34 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block content %}
+
+<div class="card">
+  <div class="card-body">
+
+    <h2 class="card-title">Edit Survey</h2>
+    <br />
+
+    <table class="table table-hover table-striped card-text">
+    <thead>
+      <tr>
+        <th scope="col">Survey Name</th>
+        <th scope="col">Created</th>
+        <th scope="col">Updated</th>
+      </tr>
+    </thead>
+    <tbody>
+      {% for survey in surveys %}
+        <tr>
+          <td><b><a href="/fobi/forms/edit/{{survey.id}}">{{survey.name}}</a></b></td>
```

Comment:
Yes! I agree and was planning to do this—I'm not sure why Fobi uses these labels as identifiers instead of something like a UUID, and may switch it over.

Reply:
Great. Do you think you could do that in this PR?

Reply:
Yep!